docs/CHANGELOG.asciidoc

```diff
@@ -90,6 +90,7 @@
 * Improve the initialization of seasonal components for anomaly detection. (See
   {ml-pull}1201[#1201], issue: {ml-issue}#1178[#1178].)
 * Fix possible root cause for "Bad variance scale nan" log errors. (See {ml-pull}1225[#1225].)
+* Change instrumentation timestamp resolution to millisecionds. (See {ml-pull}1237[#1237].)
```

Comment:
We don't need to release note this for both 7.7.1 and 7.8.0 because 7.7.1 will be released first.

Reply:
Thanks, I updated the changelog.